### DISCUSSION TITLE
Open Call for New Feature Requests

### DISCUSSION THREAD

**Discussion by littlespex** [streaming-video-technology-alliance/common-media-library#54] [Ideas]
We are looking to expand the Common Media Library with more useful utilities. If you have an idea for a new feature, leave a comment below.

**Comment by PJSP** [comment]
There's a lot possible around subtitles and parsing and rendering there: CEA607/708 parsing (and even rendering), TTML, WebVTT, ... It's all stuf we all implemented multiple times already, but issues in any of these can be quite painful. Especially if you merge this with accessibility requirements to modify fonts, colours, ...
Getting some of that data would require CMAF/TS parsing as well as NAL parsing, but I assume we're all implementing the same thing there as well. For the parsing part I would suggest to write this in RUST or a similar language to allow for compilation to WebAssembly.
